app/routes/events/view/sessions/list.js

```diff
@@ -55,7 +55,6 @@ export default Route.extend({
     }
 
     let queryObject = {
-      include      : 'speakers',
```

Comment:
how will this remove only the `deleted` speakers? This will stop including speakers at all. Also, deleted speakers should never be returned by the server in the first place unless `get_trashed=true` is set. Investigate.